utils/core/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -71,9 +68,9 @@ class OrtAuthenticator(private val original: Authenticator? = null) : Authentica
         }
     }
 
-    // TODO: Add support for ".authinfo" files (which use the same syntax as .netrc files) once Git.kt does not call the
-    //       Git CLI anymore which only supports ".netrc" (and "_netrc") files.
-    private val netrcFileNames = listOf(".netrc", "_netrc")
+    // First look for (potentially machine-specific) credentials in a netrc-style file, then look for generic
+    // credentials passed as environment variables.
+    private val delegateAuthenticators = listOf(NetRcAuthenticator(), EnvVarAuthenticator())
 
     private val serverAuthentication = mutableMapOf<String, PasswordAuthentication>()
```

Comment:
This has not been changed by this PR, but storing the credentials in a plain map is not thread-safe.

Reply:
Changed in a new commit.